wagtail/contrib/search_promotions/forms.py

```diff
@@ -45,12 +61,29 @@ class Meta:
 
 
 class SearchPromotionsFormSet(SearchPromotionsFormSetBase):
-    minimum_forms = 1
-    minimum_forms_message = _(
+    """
+    Form set for search promotions
+
+    Attributes
+
+        - minimum_forms (int): Number of forms required for this set.
+    """
+
+    REQUIRED_FIELDS_MISSING = _("You must recommend a related page OR an external url.")
```

Comment:
Please update all user facing messages to use `URL` not `url`, this is how we refer to URLs in all other messaging currently.

However, it's probably better to consistently refer to these as 'external links', and then the inner parts are either `URL` or `Title`. Please update to something like this...

```
_("You must recommend a related page OR an external link.")
``